experiment/simulation/index.html

```diff
@@ -1,13 +1,226 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Representation of Integers and their Arithmetic</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-		<!-- Your code goes here-->
+    <div class="container justify-content-center">
+        <div class="grid">
+            <div class="row mx-4 my-3 my-sm-3 d-flex flex-row row-10 m-auto px-4">
+                <h1 id="main-heading">Integer Arithmetic</h1>
+            </div>
+            <div class="row">
+                <div class="message" id="message" value="message"></div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-md-5">
+                <div class="buttons">
+                    <span class="br"></span>
+                    <button onclick="addFunction()" class="btn">Add</button>
+                    <span class="br"></span>
+                    <button id="four-reset" onclick="reset(event)" class="btn">RESET</button>
```

Comment:
Why is this button in all capitals? Please use consistent labelling unless there is a specific reason.